theano/tensor/nnet/abstract_conv.py

```diff
@@ -66,6 +66,8 @@ def get_conv_output_shape(image_shape, kernel_shape,
     """
     bsize, imshp = image_shape[0], image_shape[2:]
     nkern, kshp = kernel_shape[0], kernel_shape[2:]
+    if not filter_dilation:
```

Comment:
Change this condition to "if filter_dilation is not None", in case someone put 0, I think it will get converted to a tuple with 0s in them. Just to be sure to not cause different error for 0s handling.